packages/fluentui/accessibility/src/behaviors/Radio/radioGroupItemBehavior.ts

```diff
@@ -29,6 +31,12 @@ const radioGroupItemBehavior: Accessibility<RadioGroupItemBehaviorProps> = props
       },
     },
   },
+
+  focusZone: {
+    props: {
+      shouldEnterInnerZone: event => keyboardKey.getCode(event) === keyboardKey.Tab,
```

Comment:
why is this needed? as far as I remember, radios do not use focus zone

Reply:
Reverted